modules/TestUtils.js

```diff
@@ -20,6 +20,12 @@ export const isObject = (object) =>
   object && !isArray(object) && typeof object === 'object'
 
 /**
+ * Returns true if the given object is an empty object.
+ */
+export const isEmptyObject = (object) =>
+  isObject(object) && Reflect.ownKeys(object).length === 0
```

Comment:
Sure. I will use `Object.getOwnPropertyNames()` and `Object.getOwnPropertySymbols()` instead